spyder/plugins/completion/providers/languageserver/conftabs/linting.py

```diff
@@ -89,6 +94,43 @@ def __init__(self, parent):
         configuration_options_group = QGroupBox(_("Provider options"))
         configuration_options_layout = QVBoxLayout()
 
+        # ruff options
+        self.ruff_exclude = self.create_lineedit(
+            _("Exclude these files or directories:"),
+            'ruff/exclude',
+            alignment=Qt.Horizontal,
+            word_wrap=False,
+            placeholder=_("Exclude test files: (?!test_).*\\.py"),
+        )
+
+        ruff_select = self.create_lineedit(
+            _("Show these errors or warnings:"),
+            'ruff/extendSelect',
+            alignment=Qt.Horizontal,
+            word_wrap=False,
+            placeholder=_("Example codes: E113, W391"),
+        )
+
+        ruff_ignore = self.create_lineedit(
+            _("Ignore these errors or warnings:"),
+            'ruff/extendIgnore',
+            alignment=Qt.Horizontal,
+            word_wrap=False,
+            placeholder=_("Default is: D,DOC,E,W,C90"),
```

Comment:
```suggestion
            placeholder=_("Default is: E"),
```

I re-read more carefully Ruff docs and at the beginning of the Rules section they say:

> By default, Ruff enables Flake8's F rules, along with a subset of the E rules

So we only need to ignore the `E` ones.